File: php/processCsv.php
<?php
$filename = "numbers.csv";
$file = fopen($filename, "r");

$numbers = explode(',', fread($file, filesize($filename)));
$counter = 0;
foreach($numbers as $value) {
    $isBig = $value >= 50 ? 1 : 0;
    echo $value . ' ' . $isBig . "\n";
    $counter += $isBig;
}

echo "\nCount of big numbers is " . $counter . "\n";
?>

File: python/processCsv.py
try:
  with open('numbers.csv', 'r') as file:
    numbers = list(map(int, file.readline().split(',')))
except:
  print("File does not exist")
  
counter = 0
for number in numbers:
  isBig = 0 if number <= 50 else 1
  print(number, isBig)
  counter += isBig
print("Count of big numbers is", counter)